LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020-2021 PostHog Inc.
+Copyright (c) 2022-present PostHog Inc.
```

Comment:
Oh yeah that's outdated! 

Just clarifying internally if "present" is acceptable, or if we just want the current year.
Also if we do `20XX-present` then this would need to be `2020-present`